libs/perun/services/src/lib/entity-storage.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { Facility, Group, Resource, Vo } from '@perun-web-apps/perun/openapi';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EntityStorageService {
+
+   entity: Vo | Group | Facility | Resource;
+
+   setEntity(entity: Vo | Group | Facility | Resource) {
+     this.entity = entity;
+   }
+
+  getEntity(): Vo | Group | Facility | Resource {
+     return this.entity;
+   }
```

Comment:
also plese provide some comment what purpouse have this component, that we only want to read the ids and bean names from this objects and that we don't want to store in this objects names and things that can change